docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,28 +1,33 @@
 
 == New Features
 
-- Added LDAP Security Provider support for Management Center CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/892[#892])
+- Tiered Storage support for Hazelcast and Map CRs (https://github.com/hazelcast/hazelcast-platform-operator/pull/984[#984])
+- CP Subsystem configuration support for Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/1041[#1041])
+- Full and Delta WAN Sync support (https://github.com/hazelcast/hazelcast-platform-operator/pull/977[#977])
+- Added possibility to configure `consistencyCheckStrategy` in WanReplication CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/1072[#1072])
+- Introduced localConfig was added to Persistence configuration to restore from existing PVs which are created by the Helm chart or the Operator (https://github.com/hazelcast/hazelcast-platform-operator/pull/1043[#1043])
+- Exposed ingress path for external connectivity in the Management Center CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/989[#989])
+- Added Custom Query Attributes into Map CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/978[#978])
 
 == Changes
 
-- Added the ability to configure Interfaces for ClientServer (https://github.com/hazelcast/hazelcast-platform-operator/pull/920[#920])
-- The new `hazelcastendpoint` Custom Resource Definition (CRD) provides a better way to list the external addresses of your Hazelcast cluster (https://github.com/hazelcast/hazelcast-platform-operator/pull/860[#860])
-- A `hazelcast-resource` column has been added to the `additionalPrinterColumns` for the wide output of CRDs referencing Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/899[#899])
-- Failed HotBackup upload to bucket now correctly handles partially uploaded files and tries to remove them (https://github.com/hazelcast/hazelcast-platform-operator/pull/885[#885])
-- The `v.Err` function was moved to `fieldValidator` to remove duplication (https://github.com/hazelcast/hazelcast-platform-operator/pull/886[#886])
-- Improved validations for HotBackup, WANReplication, JetJob, JetJobSnapshot, Map, Cache, and Topic (https://github.com/hazelcast/hazelcast-platform-operator/pull/877[#877])(https://github.com/hazelcast/hazelcast-platform-operator/pull/876[#876])(https://github.com/hazelcast/hazelcast-platform-operator/pull/875[#875])(https://github.com/hazelcast/hazelcast-platform-operator/pull/874[#874])(https://github.com/hazelcast/hazelcast-platform-operator/pull/873[#873])(https://github.com/hazelcast/hazelcast-platform-operator/pull/872[#872])(https://github.com/hazelcast/hazelcast-platform-operator/pull/871[#871])(https://github.com/hazelcast/hazelcast-platform-operator/pull/859[#859])
+- Removed the ability to configure `persistence.baseDir` field for user convenience (https://github.com/hazelcast/hazelcast-platform-operator/pull/1028[#1028])
```

Comment:
```suggestion
- Removed the ability to configure `persistence.baseDir` field (https://github.com/hazelcast/hazelcast-platform-operator/pull/1028[#1028])
```

Reply:
it read as though we had removed the field that was convenient for the user.

If you want tokeep this, I assume youmean we have made it more convenient, so alternative suggestion below.

Reply:
```suggestion
- To simplify your configuration, removed the ability to configure `persistence.baseDir` field (https://github.com/hazelcast/hazelcast-platform-operator/pull/1028[#1028])
```

Reply:
@rebekah-lawrence I checked the PR one more time. What if I'll keep like this: 

"- Removed the ability to configure `persistence.baseDir` field that will be managed by the operator internally(https://github.com/hazelcast/hazelcast-platform-operator/pull/1028[#1028])"